pkg/system/testing/names.go

```diff
@@ -0,0 +1,11 @@
+package testing
+
+import (
+	"os"
+
+	"github.com/knative/eventing/pkg/system"
+)
+
+func init() {
+	os.Setenv(system.NamespaceEnvKey, "knative-testing")
```

Comment:
Why not `knative-eventing`?

Reply:
Is the intention to put all knative pieces into a single namespace? Just in tests? Or in production as well?